<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>RF output power</string>
    </property>
    <widget class="QLineEdit" name="leHost_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConDis_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>16</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>GET</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConDis_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>SET</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QPushButton" name="btnConDis">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>70</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbHost">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>22</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leHost">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbPort">
     <property name="geometry">
      <rect>
       <x>229</x>
       <y>20</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePort">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Reader Info</string>
    </property>
    <widget class="QLabel" name="lbVersion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ver:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbTemp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Temp:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
